packages/flutter/lib/src/widgets/text.dart

```diff
@@ -210,16 +215,40 @@ class Text extends StatelessWidget {
     this.textScaleFactor,
     this.maxLines,
   }) : assert(data != null),
+       span = null,
        super(key: key);
 
+  /// Creates a text widget from a [TextSpan].
+  const Text.fromSpan(this.span, {
+    Key key,
+    this.textAlign,
+    this.textDirection,
+    this.softWrap,
+    this.overflow,
+    this.textScaleFactor,
+    this.maxLines,
+  }): assert(span != null),
+      data = null,
+      style = null,
+      super(key: key);
+
   /// The text to display.
+  ///
+  /// If constructed with the default constructor, `span` will be null.
```

Comment:
We usually frame this from the context of the constructors, so:

```
/// The text to display.
///
/// This will be null if a [textSpan] is provided instead.
```

Reply:
Should I be specifically referring to the names of the constructor arguments, or just their types?

Reply:
As a general rule, we refer to the names of the properties, and name the constructor arguments the same as the properties.